reports/templates/reports/report.css

```diff
@@ -0,0 +1,110 @@
+@page {
+  font-family: "{{report.layout.font}}";
+  /* meta content (see https://www.quackit.com/css/at-rules/css_top-left-corner_at-rule.cfm) */
+  @bottom-right-corner {
+    content: counter(page);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+  @top-center {
+    content: "Permis n°{{permit_request.id}}";
```

Comment:
translate

Reply:
Not understood. Translate the "Permis n°" part ?